test/testenv/util.go

```diff
@@ -528,6 +528,19 @@ func DumpGetPods(ns string) []string {
 	return splunkPods
 }
 
+// GetOperatorPod prints and returns operator pod in the namespace
```

Comment:
Duplicate method. It already exists on the appframework branch.

Reply:
Same location than in appframework branch, this should avoid merge conflict.